README.md

```diff
@@ -149,7 +149,7 @@ The component accepts the following props:
 |**`resizableColumns`**|boolean|false|Enable/disable resizable columns
 |**`customToolbar`**|function||Render a custom toolbar
 |**`customToolbarSelect`**|function||Render a custom selected rows toolbar. `function(selectedRows, displayData, setSelectedRows) => void`
-|**`customFooter`**|function||Render a custom table footer. `function(count, page, rowsPerPage, changeRowsPerPage, changePage) => string`&#124;` React Component`
+|**`customFooter`**|function||Render a custom table footer. `function(count, page, rowsPerPage, changeRowsPerPage, changePage, textLabels) => string`&#124;` React Component`
```

Comment:
How about we point out that text labels is an object or perhaps link to the `textLabels.js` file so people know how to implement it themselves if they want to change the wording?